src/libraries/System.Data.Common/src/System/Xml/XPathNodePointer.cs

```diff
@@ -2,6 +2,8 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable disable
```

Comment:
What's up with all of these #nullable disables?

Reply:
All these (under System/Xml) are because of the dependencies on System.Private.Xml, which doesn't seem to be annotated.